src/pages/ProductList/ProductList.js

```diff
@@ -17,18 +16,18 @@ const ProductList = () => {
 
   const params = useParams();
   const navigate = useNavigate();
+  const location = useLocation();
 
   useEffect(() => {
-    fetch(`IP주소/items/${params.category}/${params.subcategory}`)
+    fetch(
+      `IP주소/items/${params.category}/${params.subcategory}${location.search}`
+    )
       .then(res => res.json())
-      .then(data => setProductData(data));
-  }, []);
-
-  console.log(productData);
+      .then(data => setProductData([data]));
```

Comment:
```suggestion
      .then(data => setProductData(data));
```
배열안에 넣어서 data를 넣는 것은 불필요한 depth를 만드는 방법이라고 보입니다!